setup.py

```diff
@@ -49,11 +49,12 @@ def read(fname):
         PACKAGE_NAME: [
             'countries_regions.csv',
             'countries_regions_phase1_fix.csv',
-            'covid_xprize/examples/predictors/lstm/tests/fixtures/*',
-            'covid_xprize/standard_predictor/data/*',
-            'covid_xprize/standard_predictor/models/*',
-            'covid_xprize/validation/data',
+            'examples/predictors/lstm/tests/fixtures/*',
```

Comment:
Don't we need the `covid_xprize` folder here? How does it know where to look from the root folder?

Reply:
It's actually relative to the package. Here the package files are associated with the key `PACKAGE_NAME==covid_xprize` so it is relative to that directory. It took a bit of testing for me to figure it out, but it's also mentioned in the docs; see:

https://docs.python.org/3/distutils/setupscript.html#installing-package-data

Reply:
ok makes sense. Thanks!